website/client/components/guilds/membersModal.vue

```diff
@@ -0,0 +1,109 @@
+<template lang="pug">
+div
+  button.btn.btn-primary.btn-purple(b-btn, @click="$root.$emit('show::modal','members-modal')") {{ $t('viewMembers') }}
+
+  b-modal#members-modal(:title="$t('createGuild')")
+    ul(v-for='member in members', :key='member')
+      li(@click='clickMember') {{member}}
+        button(@click='removeMember(member)', v-once) {{$t('remove')}}
+        button(@click='quickReply(member)', v-once) {{$t('message')}}
+        button(@click='addManager(member)', v-once) {{$t('addManager')}}
+        button(@click='removeManager(member)', v-once) {{$t('addManager')}}
+
+  b-modal#remove-member(:title="$t('confirmRemoveMember')")
+    button(@click='confirmRemoveMember(member)', v-once) {{$t('remove')}}
+
+  b-modal#private-message(:title="$t('confirmRemoveMember')")
+    button(@click='confirmRemoveMember(member)', v-once) {{$t('remove')}}
+</template>
+
+<script>
+import bModal from 'bootstrap-vue/lib/components/modal';
+import bBtn from 'bootstrap-vue/lib/components/button';
+
+export default {
+  props: ['group'],
+  components: {
+    bModal,
+    bBtn,
```

Comment:
is this component used somewhere? If yes, can we use the simple button with proper css classes?